frontend/eda/Resources/credentials/hooks/useDeleteCredentials.tsx

```diff
@@ -2,7 +2,7 @@ import { useCallback, useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
 import { compareStrings, useBulkConfirmation } from '../../../../../framework';
 import { requestDelete } from '../../../../common/crud/Data';
-import { idKeyFn } from '../../../../hub/api';
+import { idKeyFn } from '../../../../hub/api/utils';
```

Comment:
It would be good to move `idKeyFn` from [frontend/hub/api/utils](https://github.com/ansible/ansible-ui/pull/786/files#diff-7a0a3bfbfe4f1fbef77ff3b622048ac41d5c689af723e357d26380e79b82d605R94) to `frontend/common/utils` if we're looking to access this util function in EDA files such as this one.